index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <h1 id="title">OUR SURVEY FORM</h1>
+  <p id="description"><i>this is survey form please fill in an appropiate manner</i></p>
+  <form id="survey-form">
+    <label id="name-label">enter your name here</label>
+    <input required type="text" id="name" placeholder="name">
+    <br><br>
+    <label id="email-label">Enter your email:</label>
+    <input required type="email" id="email" placeholder="email">
+    <br><br>
+    <label id="number-label">Enter your number:</label>
+    <input required type="number" id="number" max="50" min="10" placeholder="number">
+    <br><br>
+    <label>select from the dropdown</label>
+    <select name="dropdown" id="dropdown">
+      <option value="languge"></option>
+
+      <option value="country"></option>
+
+      <option value="time zone"></option>
+    </select>
```

Comment:
please avoid unnecessary blank space for practicing best HTML practice

Reply:
Great job being polite and giving a reason for the requested change.